ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContexts.scala

```diff
@@ -0,0 +1,382 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.infrastructure.participant
+import java.time.Instant
+
+import com.daml.ledger.api.refinements.ApiTypes.TemplateId
+import com.daml.ledger.api.testtool.infrastructure.Endpoint
+import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext.CompletionResponse
+import com.daml.ledger.api.testtool.infrastructure.time.DelayMechanism
+import com.daml.ledger.api.v1.active_contracts_service.GetActiveContractsRequest
+import com.daml.ledger.api.v1.admin.config_management_service.{
+  GetTimeModelResponse,
+  SetTimeModelRequest,
+  SetTimeModelResponse,
+  TimeModel,
+}
+import com.daml.ledger.api.v1.admin.package_management_service.{
+  PackageDetails,
+  UploadDarFileRequest,
+}
+import com.daml.ledger.api.v1.admin.participant_pruning_service.PruneResponse
+import com.daml.ledger.api.v1.admin.party_management_service.PartyDetails
+import com.daml.ledger.api.v1.command_completion_service.{
+  Checkpoint,
+  CompletionEndRequest,
+  CompletionEndResponse,
+  CompletionStreamRequest,
+  CompletionStreamResponse,
+}
+import com.daml.ledger.api.v1.command_service.{
+  SubmitAndWaitForTransactionIdResponse,
+  SubmitAndWaitForTransactionResponse,
+  SubmitAndWaitForTransactionTreeResponse,
+  SubmitAndWaitRequest,
+}
+import com.daml.ledger.api.v1.command_submission_service.SubmitRequest
+import com.daml.ledger.api.v1.commands.Command
+import com.daml.ledger.api.v1.completion.Completion
+import com.daml.ledger.api.v1.event.CreatedEvent
+import com.daml.ledger.api.v1.ledger_configuration_service.LedgerConfiguration
+import com.daml.ledger.api.v1.ledger_offset.LedgerOffset
+import com.daml.ledger.api.v1.package_service.{GetPackageResponse, PackageStatus}
+import com.daml.ledger.api.v1.transaction.{Transaction, TransactionTree}
+import com.daml.ledger.api.v1.transaction_service.{
+  GetTransactionByEventIdRequest,
+  GetTransactionByIdRequest,
+  GetTransactionsRequest,
+  GetTransactionsResponse,
+}
+import com.daml.ledger.api.v1.value.{Identifier, Value}
+import com.daml.ledger.client.binding.{Primitive, Template}
+import com.daml.lf.data.Ref.HexString
+import com.google.protobuf.ByteString
+import io.grpc.health.v1.health.HealthCheckResponse
+import io.grpc.stub.StreamObserver
+
+import scala.concurrent.Future
+
+trait ParticipantTestContext extends UserManagementTestContext {
```

Comment:
Can you align the type with the file name?

Reply:
The problem is that if I do that it messes the git history, as it aligns the content of this file with the old one, so in the history, it will appear as though I deleted all the implementations. I was thinking of doing the align in another PR, which should help keep the history. Wdyt?

Reply:
Didn't think of that. Another commit in this PR won't do the trick because they're eventually squashed? If not, I'm fine with doing it in another PR.

Reply:
If it's another commit here, github still sees only the squashed output of it. hopefully by doing it in another PR this will be fixed